### PR TITLE
Update readme for Upbound Crossplane

### LINKED ISSUE #15: Update Readme
### What problem are you facing?

Readme is same as crossplane/crossplane right now. We need to update it.

### How could Crossplane help solve your problem?

Update readme with enough details on what this repo is about.

### PR DESCRIPTION
**Pull request by turkenh**
### Description of your changes

Updates readme document with the purpose of the repo and links for further details.

Fixes #15 

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

N/A

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,14 @@
-![CI](https://github.com/crossplane/crossplane/workflows/CI/badge.svg) [![GitHub release](https://img.shields.io/github/release/crossplane/crossplane/all.svg?style=flat-square)](https://github.com/crossplane/crossplane/releases) [![Docker Pulls](https://img.shields.io/docker/pulls/crossplane/crossplane.svg)](https://img.shields.io/docker/pulls/crossplane/crossplane.svg) [![Go Report Card](https://goreportcard.com/badge/github.com/crossplane/crossplane)](https://goreportcard.com/report/github.com/crossplane/crossplane) [![Slack](https://slack.crossplane.io/badge.svg)](https://slack.crossplane.io) [![Twitter Follow](https://img.shields.io/twitter/follow/crossplane_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=crossplane_io&user_id=788180534543339520)
+# Upbound Crossplane
 
-![Crossplane](docs/media/banner.png)
+This is the [Upbound] maintained version of [Crossplane] which is bundled and
+shipped as part of the [Universal Crossplane] project.
```
Comment (luebken) [inline README.md:4]: In my lingo, UXP is rather a product than a project. Without the "project" it sounds better to me.

**Review by luebken — COMMENTED** [review]
I would like to have @grantgumina have a look. This could be a good opportunity to describe how these different projects & products tie in together.

File: README.md (hunk 1 above)
Comment (grantgumina) [reply README.md:4]: Yes, I'd rather call it a project as well since it will (eventually) have it's own community form around it (or at least we think).

**Review by grantgumina — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (grantgumina) [reply README.md:4]: I was going to reveal this during the all-hands, but we've been working on an updated messaging framework which should help with these kinds of things in the future. https://docs.google.com/spreadsheets/d/1gj4F3L4sO7k7UbyAYcS2cxhWgRb6_fOsDe4i-GFNVF4/edit#gid=688587790

**Pull request merged**